database/migrations/2021_09_13_183508_create_blog_subcategories_table.php

```diff
@@ -14,7 +14,7 @@ class CreateBlogSubcategoriesTable extends Migration
     public function up()
     {
         Schema::create('blog_subcategories', function (Blueprint $table) {
-            $table->primary('id');
+            $table->z('id');
```

Comment:
Could you please tell me, what does z works here

Reply:
Sorry, I will fix it soon :)